compile.sh

```diff
@@ -329,7 +331,10 @@ if ! [ -f src/wlapplication.cc ] ; then
   exit 1
 fi
 
-bzr pull
+# Checkout master and pull latest version
+git checkout master
+git pull https://github.com/widelands/widelands.git master
```

Comment:
Here we should use
```
git pull --rebase https://github.com/widelands/widelands.git master
```

Reply:
My experience with rebase is that sometimes you have to resolve the same conflict again and again and again during 1 rebase process if there are multiple commits involved. And if you don't understand them, you have to abort, which means learning how to abort this on the command line. This make the learning curve far too high, so I think merging would be easier here.